### DISCUSSION TITLE
Is there a way to make a spawned vehicle my personal vehicle?

### DISCUSSION THREAD

**Discussion by MMMash** [YimMenu/YimMenu#1587] [Q&A]
Title says it all. I wanna have certain cars that aren't readily available but that I can spawn in.

**Comment by Marlodor96** [comment]
u can already spawn unavailable cars and put into your garage.

**Comment by LintStalker** [comment]
Yes, drive a car off the street, into one of your regular garages, and before you actually get into the garage, bring up Yimmenu and spawn the car that you want.  You can just do the base model, and then make changes to it once it's yours.

Afterwards, you might want to drive it into a LCS or your autoshop and check to make sure it has insurance.

**Reply by Marlodor96** [reply]
In the latest update when u enter in the car from garage it will autoadd insurance

**Reply by LintStalker** [reply]
oh,  that's cool!  Thanks for letting me know!

**Reply by chenry0553** [reply]
I've been trying this but haven't had success yet. I've got to be missing something. Do you have to check spawn inside car? if you have any other specific steps I need to follow for this to work for me, that would be great.

**Reply by LintStalker** [reply]
Yes, you have to check spawn inside car.  You can't use a high end garage either.  One of the 2,6 or 10 should work.

**Reply by chenry0553** [reply]
Sorry, last question...Does the garage have to be full? Maybe my timing is off on when to click on the vehicle I want to spawn in. After entering the garage its the same street car I went in with, and when I drive outside the garage I see the car I wanted to spawn inside on the roof or right outside the garage door. nvm got it ty, just timing :)

**Comment by LintStalker** [comment]
No, it doesn’t have to be full.

I edge up to the garage and when it shows me driving in, I spawn the car I
want.

**Comment by Ronu19** [comment]
Yes i tried and here is my trick to make spawn vehicle owner.

session:- invite only

note:- make sure your first list of garage have a space left to get your car

> spawn the vehicle & mod it as you like 

> then hit  "start LS Custom' option inside yimmenu vehicle section

> choose Loss/Theft Prevention

> now buy the "Tracker" & get out of the car

> switch to Franklin

> go online again & call the mechanic

> your car is in your first garage list 🤌
